package/boot/uboot-envtools/files/mvebu

```diff
@@ -26,6 +26,7 @@ glinet,gl-mv1000)
 	;;
 globalscale,espressobin|\
 globalscale,espressobin-emmc|\
+globalscale,espressobin-ultra|\
```

Comment:
Technically, you should bump PKG_RELEASE for uboot-envtools when changing the content.

Reply:
Missed that one, should I split the this part and make a separate PR with PKG_RELEASE bump?
From what I can see, that's how it's usually done but would just like to check first.

Reply:
Not necessarily, that's essentially matter of taste, different people do it differently. As you prefer ...